### PR TITLE
Feature/table arrow key events

### PR DESCRIPTION
**Pull request by lab900-senn-lambrecht**

### CODE REVIEW

File: lib/src/lib/table/cell-editors/cell-select-editor/cell-select-editor.component.ts [hunk 1]
```diff
@@ -66,21 +60,33 @@ export class CellSelectEditorComponent extends CellEditorAbstract<CellSelectEdit
...
+  private _matSelect?: MatSelect;
 
-  public readonly defaultCompareFn = (a: any, b: any): boolean => a === b;
+  @ViewChild(MatSelect)
+  private set matSelect(value: MatSelect) {
+    this._matSelect = value;
+    setTimeout(() => {
+      if (this._matSelect && !this._matSelect.panelOpen) {
+        this._matSelect.focus();
+        this._matSelect.open();
```
Comment (lab900-gino-vanhoudt) [inline lib/src/lib/table/cell-editors/cell-select-editor/cell-select-editor.component.ts:75]: doesn't this mean that all mat selects will open automatically on initialisation of the component?

File: lib/src/lib/table/components/table-cell/table-cell.component.ts [hunk 2]
```diff
@@ -131,19 +135,16 @@ export class Lab900TableCellComponent<T = any> implements OnDestroy {
...
+    this.tableService.sort$;
 
   public readonly defaultCellRenderer = DefaultCellRendererComponent;
   public readonly defaultHeaderRenderer = DefaultColumnHeaderRendererComponent;
 
   @Input({ required: true })
   public trackByTableFn!: TrackByFunction<T>;
 
-  public constructor(
-    @Optional() @SkipSelf() public table: MatTable<any>,
```
Comment (lab900-gino-vanhoudt) [inline lib/src/lib/table/components/table-cell/table-cell.component.ts:143]: what did @skipself do? and did you account for that?

**Review by lab900-gino-vanhoudt — APPROVED** [review]

File: lib/src/lib/table/components/table-cell/table-cell.component.ts (hunk 2 above)
Comment (lab900-senn-lambrecht) [reply lib/src/lib/table/components/table-cell/table-cell.component.ts:143]: This was for the compiler, it complained about circular dependency risk but this didn't make sense. It doesn't complain anymore with inject()

File: lib/src/lib/table/cell-editors/cell-select-editor/cell-select-editor.component.ts (hunk 1 above)
Comment (lab900-senn-lambrecht) [reply lib/src/lib/table/cell-editors/cell-select-editor/cell-select-editor.component.ts:75]: This was already the case but done differently

**Pull request merged**